### DISCUSSION TITLE
audio-visual simultaneous presentation

### DISCUSSION THREAD

**Discussion by mecui22** [jspsych/jsPsych#1226] [Q&A]
hi - is there a plug-in for simultaneous presentation of audio and visual stimulus?

For example, participants hear the word "apple" while seeing the word "apple" written on the screen.

Thank you!!

**Comment by jodeleeuw** [comment]
Hi @mecui22,

You can use the audio plugins to do this. The prompt parameter can display the word (or any HTML that you'd like).

**Reply by mecui22** [reply]
good suggestion @jodeleeuw 

Here is a follow-up: can I change the stimulus duration of the visual prompt, e.g., the audio cue lasts for 5 s, while the visual stimulus only stays for 1 sec.

Many thanks!

**Reply by jodeleeuw** [reply]
You could add this feature in by modifying the plugin. Here's [a thread](https://github.com/jspsych/jsPsych/discussions/1191) where someone is adding a `stimulus_duration` parameter to control the timing.

**Reply by kurokida** [reply]
@mecui22 
Please consider to use the jspsych-psychophysics plugin I made. 
https://jspsychophysics.hes.kyushu-u.ac.jp/
